h/static/scripts/group-forms/components/CreateEditGroupForm.tsx

```diff
@@ -253,7 +277,39 @@ export default function CreateEditGroupForm() {
           classes="h-24"
         />
 
-        <div className="flex items-center gap-x-4">
+        {config.features.group_type && (
+          <>
+            <Label id={groupTypeLabel} text="Group type" />
+            <RadioGroup<GroupType>
```

Comment:
The `RadioGroup` component is parametrized by the type of value. In some cases this can be inferred (eg. from the value passed to `selected`), but here it needed to be specified.